docs/install/install-knative-gcp.md

```diff
@@ -72,23 +72,16 @@ information about Workload Identity, please see
 1. Your gcloud `CLI` are up to date. You may use `gcloud components update` to
    update it.
 
-**_Note_**: Both scripts will have a step to create a Google Cloud Service
-Account `cloud-run-events`. Ignore the error message if you already had this
-service account (error for 'service account already exists').
-TODO([#896](https://github.com/google/knative-gcp/issues/896)) Get rid of the
```

Comment:
Let's also delete the TODO here: https://github.com/google/knative-gcp/blob/master/docs/install/install-knative-gcp.md

Reply:
The one I deleted is the same as the one you linked, right?

Reply:
Sorry my bad, for some reason I thought I added the TODO in 2 places. Thankfully I didn't :)